src/capitalsFirst.js

```diff
@@ -18,7 +18,19 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let initialArray = str.split(' ');
+  let uppercaseLetterFirst = [];
+  let lowercaseLetterFirst = [];
```

Comment:
you dont need let if you dont reassign a variable [prefer-const rule](https://mate-academy.github.io/style-guides/javascript.html#variables--const)
All 3 above